winpr/libwinpr/file/file.c

```diff
@@ -253,7 +253,7 @@ static BOOL FileLockFileEx(HANDLE hFile, DWORD dwFlags, DWORD dwReserved,
 		DWORD nNumberOfBytesToLockLow, DWORD nNumberOfBytesToLockHigh,
 		LPOVERLAPPED lpOverlapped)
  {
-	int lock;
+	int lock = 0;
```

Comment:
lock is always set a few lines below, so I don't think the initialization is useful